Monika After Story/game/script-topics.rpy

```diff
@@ -4790,6 +4790,299 @@ label monika_urgent:
+	m "Or you could, you know, spend some time with me while you wait ehehe~"
+	Menu:
+		m "Do you know about the stuff the people that run these places do to make lines more bearable?"
+		"No":
+			m "Ah, I guessed so."
+			m "Most people don't notice the less obvious stuff, like a merchandise store near a line for example."
+			m "But then there's more obvious stuff, like decorating the place in an amusement park as an entrance to what
+			you're waiting for!"
+			m "Any small details that can distract the people waiting is a benefit for both sides!"
+			m "It means more revenue for one side in the case of merchandise and less boredom for the consumer."
+			m "Of course banks and similar places don't do stuff like this!"
+			m "They don't need to make sure people stay entertained since people need to use that place, so they'll just have to deal with it."
+			m "Next time, msybe you can think of me while you're waiting in line ehehe~"
+		"Yes":
+			m "Gosh, I'm pretty surprised then!"
+			m "I didn't expect you to know or realize honestly."
+			m "Since you already know about that, why don't I suggest some stuff to maybe pass your time!"
+			m "You can learn a lot about the things that pass by, such as people or cars!"
+			m "Ah, that gets boring fast though!"
+			m "Let me think ehehe~"
+			m "..."
+			m "Oh, I got it!"
+			m "If you're a creative person you can inspire yourself around what you see!"
+			m "If you're not that creative, talk to the people in line."
+			m "You can always make friends in the unlikeliest of places!"
+			m "Of course, I'll always be your loved one [player]."
+			m "I don't have that many more suggestions that wouldn't require you to bring something."
+			m "You can always bring a book or your phone!"
+			m "Try to be careful about them getting stolen though."
+			m "Perhaps you could spend some time thinking about how madly in love we are with each other while you wait next time~"
+	return
+
+init 5 python:
+    #This is a list of keywords for this topic
+    for key in ['leaf', 'leaves']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_leaf') #id
+    monika_random_topics.append('monika_leaf')
+	
+label monika_leaf:
+	m "Have you ever paid attention to the leaves around you [player]?"
+	m "You may have gotten frustated at them falling during autumn."
+	m "This might sound a bit silly to you, but I feel quite bad for those poor leaves."
+	m "Specifically, I mean leaves from deciduous plants."
+	m "These kind of plants lose their leaves to survive harsh weathers, while different sorts of plants do the opposite."
+	m "Those other types of plants are called evergreen and don't need to lose their leaves to survive."
+	m "I know leaves don't have feelings after all, but can you imagine how they must feel?"
+	m "That feeling of love and care sinking into pain and betrayal!"
+	Menu:
+		m "Can you even begin to imagine how much a betrayal like that must hurt?"
```

Comment:
Your dialogue/code in this `menu` block have an extra level of indentation. Please remove one level of indentation from **all** the code in this `menu` block.